Jenkinsfile

```
@@ -17,7 +17,21 @@ pipeline {
         JAVA_HOME = '/opt/jdk-17/'
       }
       steps {
-        sh './mvnw -V verify checkstyle:checkstyle spotbugs:spotbugs -Dmaven.test.failure.ignore -Dcheckstyle.failOnViolation=false -Dspotbugs.failOnError=false'
+        script {
+          infra.withArtifactCachingProxy() {
+            def OPTS = env.MAVEN_SETTINGS ? "-s ${MAVEN_SETTINGS}" : ''
+            sh """
+              ./mvnw -V \
+                ${OPTS} \
```

```suggestion
                --no-transfer-progress \
                ${OPTS} \
```
Nit, for shorter logs